smpp/smpptest/logger.go

```diff
@@ -0,0 +1,31 @@
+package smpptest
+
+import (
+	"fmt"
+	"log"
+	"os"
+)
+
+var (
+	Info    *log.Logger
```

Comment:
Global variables in tests is a bad idea. I like the semantics of net/http better, where you can pass an optional logger to the Server struct.